### PR TITLE
fix(w3c/headers): error when license is unknown

### LINKED ISSUE #3805: `license: "document"` and `doJsonLd: true` causes a silent error
## Important info

- URL to affected spec: https://w3c.github.io/aria/ (You need to run the source since it's prebuilt)
- ReSpec version: 26.13.4
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

**What happened (e.g., it crashed)?:**

```
TypeError: can't access property "url", conf.licenseInfo is undefined
    addJSONLDInfo http://localhost:64321/src/w3c/seo.js:94
    run http://localhost:64321/src/w3c/seo.js:67
    executeRunPass http://localhost:64321/src/core/base-runner.js:73
    executeRunPass http://localhost:64321/src/core/base-runner.js:60
    runAll http://localhost:64321/src/core/base-runner.js:29
    run http://localhost:64321/src/respec.js:13
    async* http://localhost:64321/profiles/w3c.js:76
    promise callback* http://localhost:64321/profiles/w3c.js:76
```

**Expected behavior (e.g., it shouldn't crash):**

This shouldn't happen

**Optional, steps to reproduce:**

1. Add `license: "document"` and `doJsonLd: true` to examples/basic.html
2. Run it

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #3805

### CODE REVIEW

**Review by saschanaz — APPROVED** [review]
👍

File: src/w3c/headers.js [hunk 1]
```diff
@@ -301,6 +301,15 @@ export async function run(conf) {
     const hint = docLink`Please set ${"[license]"} to "w3c-software-doc" instead.`;
     showError(msg, name, { hint });
   }
+  if (!licenses.has(conf.license)) {
+    const msg = `The license "\`${conf.license}\`" is not supported.`;
```
Comment (saschanaz) [inline src/w3c/headers.js:305]: Not recognized? No strong opinion though.

Comment (marcoscaceres) [reply src/w3c/headers.js:305]: Will keep it as is for now.

**Comment by marcoscaceres** [comment]
Thanks for the speedy review!

**Pull request merged**